release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -477,6 +477,16 @@ You can also customize the pod lifetime value for the `LifecycleAndUtilization`
 
 For more information, see xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-descheduler[Evicting pods using the descheduler].
 
+[id="ocp-4-9-nodes-multiple-logins"]
+==== Multiple logins to the same registry
+
+When configuring the `docker/config.json` file to allow pods to pull images from private registries, you can now list specific repositories in the same registry, each with credentials specific to that registry path. You can also now define a registry with a specific namespace.
+
+[id="ocp-4-9-nodes-enhanced-monitoring"]
+==== Enhanced monitoring of node resources
+
+Node-related metrics and alerts have been enhanced to give you an earlier indication of when the stability of a node is compromised, by reducing the time between the notification and the node failure.
```

Comment:
My first thought on reading "reducing the time between the notification and the node failure" was that it sounds like the opposite of what I'd want, because i was thinking the alert would happen first and then the failure (so more lead time would be better). But I'm thinking this alert is _after_ the failure, right? I wonder if it would make sense to flip the order of these two things in the sentence so the temporal relationship is really obvious.

Reply:
Thanks for pointing this out. I will look in the Jiras again to confirm.

Reply:
@jeana-redhat I am going to remove that phrase. I think it's largely redundant. I got that wording from the description in the [dev Jira](https://issues.redhat.com/browse/OCPNODE-520).

Reply:
ah yes - good point, it's totally fine without it at all :)